hypervisor/boot/acpi.c

```diff
@@ -100,7 +100,7 @@ struct acpi_madt_local_apic {
 	uint32_t                       lapic_flags;
 };
 
-static void *global_rsdp;
+static struct acpi_table_rsdp *global_rsdp;
```

Comment:
Why this is called as "global" ?  It is confusing for a static variable to be named as global.

Reply:
@dongyaozu,  thanks, have updated, pls review.